teps/0115-tekton-catalog-git-based-versioning.md

```diff
@@ -174,6 +174,10 @@ is defined in this TEP. A Catalog may contain one or more resources.
 * **Resource**: Item shared in a Catalog e.g. `Task` or `Pipeline`.
 
 * **Hub**: User interface for one or more Catalogs.
+    * **Artifact Hub (https://artifacthub.io/)**: The primary and suggested user interface for Tekton Catalogs. Catalogs with git-based versioning will **ONLY** be supported in the Artifact Hub in the first iteration of this TEP.
```

Comment:
I am not sure I follow here 🤔. Artifact Hub supports both folder-based and git-based versionning already. The tekton org doesn't control the Artifact Hub, so we cannot really say "supported" at least (discussed maybe ?).

Reply:
To make sure my understanding is correct, you are suggesting something like "we have discussed with the Artifact Hub community, and they agreed (and helped) to support the git-based versioning" in the Artifact Hub?

Reply:
Updated to "discussed" 😁